build.zig

```diff
@@ -51,7 +59,9 @@ pub fn build(b: *Build) void {
     sig_mod.addImport("httpz", httpz_mod);
     sig_mod.addImport("zstd", zstd_mod);
     sig_mod.addImport("curl", curl_mod);
-    sig_mod.addImport("rocksdb", rocksdb_mod);
+    if (blockstore_db == .rocksdb) sig_mod.addImport("rocksdb", rocksdb_mod);
```

Comment:
Looking back at this comment, I still think this should be a switch, but for a different reason. Saying "so it won't be missed" doesn't make sense, since obviously it won't be missed. Nothing would work otherwise.
My new reason for it is that I just think a switch looks better. It also shows that you don't need to have two db imports at the same time.

Reply:
eaac69fd0355df004e33c20b21c9d0f2ae13a6f2

Reply:
Perfect. I'd still like to use lmdb as a lazy dependency, however, that can easily be a follow-up enchantment.